tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -38,7 +38,7 @@
 from mlflow.utils import mlflow_tags
 from mlflow.utils.file_utils import TempDir
 from mlflow.utils.uri import extract_db_type_from_uri
-from tests.resources.db.initial_models import Base as InitialBase
+from mlflow.store.tracking.dbmodels.initial_models import Base as InitialBase
```

Comment:
Addressed [this TODO](https://github.com/mlflow/mlflow/pull/4180/files#diff-7e8f3a314f7ba34010ac4d0d42dd1e9714e10c6f7a82158b3c929bb7709521b4L1858)